content/en/docs/tasks/administer-cluster/safely-drain-node.md

```diff
@@ -101,7 +101,10 @@ afterwards to tell Kubernetes that it can resume scheduling new pods onto the no
 ## Draining multiple nodes in parallel
 
 The `kubectl drain` command should only be issued to a single node at a
-time. However, you can run multiple `kubectl drain` commands for
+time, because as the maintenance on the node is under process the pods are safely evicted 
+until the given node is in the service again and if the command is used for more than a single node, the pods
+that are part of the service may also get terminated due to eviction which can cause disruption and data loss in the running application.
```

Comment:
This is not simple English.
Can we revise it into something people can easily understand?

Reply:
@tengqm can you please mention the exact statement which is to be reviewed and changed

Reply:
Please try revise the text so that the long sentence is broken into several shorter ones. For example, when you say "an apple has fallen from the tree and it is raining",
you are supposed to break it into two sentences. We welcome texts that are easy to
parse. No one can read the whole sentence and get what you are trying to say without
having to reread it again. That is bad documentation.

Reply:
Sorry if it was not satisfactory. I will definitely come up with a better solution